src/eventsub/channel/moderator/add.rs

```diff
@@ -0,0 +1,104 @@
+#![doc(alias = "channel.moderator.add")]
+//! a user is given moderator privileges on a specified channel.
+
+use super::*;
+/// [`channel.moderator.add`](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelmoderatoradd): a user is given moderator privileges on a specified channel.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "typed-builder", derive(typed_builder::TypedBuilder))]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct ChannelModeratorAddV1 {
+    /// The broadcaster user ID for the channel you want to get moderator addition notifications for.
```

Comment:
I feel like this can be worded better

Reply:
What about "The user ID for the channel you want to get moderator addition notifications for."? Or maybe even drop the "you want to ..." part? The text from the [Twitch docs](https://dev.twitch.tv/docs/eventsub/eventsub-reference/#channel-moderator-add-condition).

Reply:
the part im struggling with is `moderator addition`, but it's fine, just a nit

Reply:
I used "Get notifications when a {mod/vip} is {added/removed} in this channel".